packages/collector/src/uncaught/index.js

```diff
@@ -183,11 +186,18 @@ function onUnhandledRejection(reason) {
 }
 
 function createEventForUnhandledRejection(reason) {
-  return createEvent(reason, 'An unhandled promise rejection occured in a Node.js process.', 5);
+  return createEvent(reason, 'An unhandled promise rejection occured in a Node.js process.', 5, true);
```

Comment:
Reminder for myself: create enum for severity and remove magic numbers

Reply:
Just for context: These numbers are dictated by the back end, those are not severity levels we use in `@instana/collector`. Of course we can add some kind of enum for them but the severity value we send to the agent with the event JSON payload needs to be an actual number with the values we use here.